plugins/node/opentelemetry-instrumentation-express/src/types.ts

```diff
@@ -97,4 +102,5 @@ export interface ExpressInstrumentationConfig extends InstrumentationConfig {
   /** Ignore specific layers based on their type */
   ignoreLayersType?: ExpressLayerType[];
   spanNameHook?: SpanNameHook;
+  spanAttributesHook?: SpanAttributesHook;
```

Comment:
In other instrumentations we usually prefer to use a hook that receives the `span` object itself and then the hook can call `span.setAttribute(...)` to modify the relevant attributes.

Can this pattern fit here as well? or is there a reason to implement it differently in this case?

Reply:
I'm fine using this approach, but that means changing the API for the plugin, since it would also remove `spanHookName`.

Reply:
Yes it will be a breaking change. We can also add the new hook and keep `spanHookName` for now and mark it deprecated.
@vmarchaud @dyladan  WDYT?